.github/actions/custom-actions/aarambh_production_custom/action.yml

```diff
@@ -34,6 +34,21 @@ runs:
       with:
         ssm_parameter: "PROD_IN_IWF_API_CHANNEL_ID"
         env_variable_name: "IWF_API_CHANNEL_ID"
+    - name: Set IWF_SECRET_KEY
+      uses: "marvinpinto/action-inject-ssm-secrets@latest"
+      with:
+        ssm_parameter: /production/serverless/${{inputs.account-sid}}/iwf_secret_key
```

Comment:
To clarify the standard, it is : `/${environment}/${serviceSecretBelongsToNotServiceThatIsUsingIt}/${twilioAccountSid}/${paramName}`

The first two paths are both a permission and a dynamic loading barriers. So we use them to say "this service has access to all of the the secrets for foo service under bar environment. Or "I want to load all of the secrets for foo service in bar environment using `getParametersByPath`. Using the service that is requesting the param instead of the service that the param relates to could also solve this, but may end up requiring duplication of params.

So in this case it would probably be `/production/iwf/${{inputs.account-sid}}/secret_key` if there are going to be different keys per twilio account. If it is global secret that changes based on env it would probably only be `/production/iwf/secret_key` And we haven't really defined a format for global secrets, but I would propose something like `/global/iwf/secret_key`.

Reply:
Since this is a bit of a priority fix, I'm happy to approve as is and refactor in a next iteration as well so long as we don't let the fix linger too long.

Reply:
Updated